models/stg_jira__version.sql

```diff
@@ -29,11 +27,11 @@ final as (
         name as version_name,
         overdue as is_overdue,
         project_id,
-        release_date,
+        cast(release_date as {{ dbt_utils.type_timestamp() }}) as release_date,
```

Comment:
Does it make sense to change it to `release_date_at` to align with our naming conventions? Though it sounds kind of strange

Reply:
Good call out here! Yeah that does sound a bit strange. I feel that since we already have `_date` in the name it should be good.